examples/hosts/app-integration/external-data/src/mock-service/externalData.ts

```diff
@@ -5,13 +5,35 @@
 
 import type { IEvent } from "@fluidframework/common-definitions";
 import { TypedEventEmitter } from "@fluidframework/common-utils";
-
+import { Response } from "node-fetch";
+/**
+ * Object based representation of initial task data.
+ */
+export interface TaskData {
+    [key: string]: {
+        name: string;
+        priority: number;
+    };
+};
 // prettier-ignore
```

Comment:
We can probably remove this comment now, since I don't think we care as much about what prettier does with the new (structured) data layout.